GitExtensions.sln

```diff
@@ -185,525 +185,179 @@ EndProject
-		{BD6AA2A2-997D-4AFF-ACC7-B64F6E51D181}.Release|Mixed Platforms.Build.0 = Release|Any CPU
-		{BD6AA2A2-997D-4AFF-ACC7-B64F6E51D181}.ReleaseTC|Any CPU.ActiveCfg = Release|Any CPU
-		{BD6AA2A2-997D-4AFF-ACC7-B64F6E51D181}.ReleaseTC|Any CPU.Build.0 = Release|Any CPU
-		{BD6AA2A2-997D-4AFF-ACC7-B64F6E51D181}.ReleaseTC|Mixed Platforms.ActiveCfg = Release|Any CPU
-		{BD6AA2A2-997D-4AFF-ACC7-B64F6E51D181}.ReleaseTC|Mixed Platforms.Build.0 = Release|Any CPU
 		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
-		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
 		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.Release|Any CPU.Build.0 = Release|Any CPU
-		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
-		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.Release|Mixed Platforms.Build.0 = Release|Any CPU
-		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.ReleaseTC|Any CPU.ActiveCfg = Release|Any CPU
-		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.ReleaseTC|Any CPU.Build.0 = Release|Any CPU
-		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.ReleaseTC|Mixed Platforms.ActiveCfg = Release|Any CPU
-		{27559302-F35E-4B62-A6EC-11FF21A5FA6F}.ReleaseTC|Mixed Platforms.Build.0 = Release|Any CPU
 		{92B57383-233E-46AE-8A86-6E98BF349A72}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{92B57383-233E-46AE-8A86-6E98BF349A72}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{92B57383-233E-46AE-8A86-6E98BF349A72}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
-		{92B57383-233E-46AE-8A86-6E98BF349A72}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
 		{92B57383-233E-46AE-8A86-6E98BF349A72}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{92B57383-233E-46AE-8A86-6E98BF349A72}.Release|Any CPU.Build.0 = Release|Any CPU
-		{92B57383-233E-46AE-8A86-6E98BF349A72}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
-		{92B57383-233E-46AE-8A86-6E98BF349A72}.Release|Mixed Platforms.Build.0 = Release|Any CPU
-		{92B57383-233E-46AE-8A86-6E98BF349A72}.ReleaseTC|Any CPU.ActiveCfg = Release|Any CPU
-		{92B57383-233E-46AE-8A86-6E98BF349A72}.ReleaseTC|Any CPU.Build.0 = Release|Any CPU
-		{92B57383-233E-46AE-8A86-6E98BF349A72}.ReleaseTC|Mixed Platforms.ActiveCfg = Release|Any CPU
-		{92B57383-233E-46AE-8A86-6E98BF349A72}.ReleaseTC|Mixed Platforms.Build.0 = Release|Any CPU
 		{9380C29C-3C7F-47A8-A4C9-A5EE9D7B386A}.Debug|Any CPU.ActiveCfg = Debug|x86
-		{9380C29C-3C7F-47A8-A4C9-A5EE9D7B386A}.Debug|Mixed Platforms.ActiveCfg = Debug|x86
 		{9380C29C-3C7F-47A8-A4C9-A5EE9D7B386A}.Release|Any CPU.ActiveCfg = Release|x86
-		{9380C29C-3C7F-47A8-A4C9-A5EE9D7B386A}.Release|Mixed Platforms.ActiveCfg = Release|x86
-		{9380C29C-3C7F-47A8-A4C9-A5EE9D7B386A}.Release|Mixed Platforms.Build.0 = Release|x86
```

Comment:
📝 Please verify that all projects are set to build under the **Any CPU** configuration. It looks like this one is not.

Also note that whatever this project is was *not* configured under either **Debug** configuration, so it should only be building for **Release|Any CPU**.

Reply:
That's the WIX setup project. It's not currently configured to build as part of a full debug or release build.

Perhaps @RussKie knows whether it should be enabled. My limited experience with these is that they're very slow to build, so perhaps we would only include it in release, if at all.

Reply:
Setup project is run via `Setup\MakeInstallers.cmd` script. It is run on the build server:
https://github.com/gitextensions/gitextensions/blob/master/appveyor.yml#L34
To prepare a release it is get invoked via `BuildInstallers.cmd` script. 

I don't really see a reason to build it at all as part of the main build, it is time consuming as pointed.

Reply:
Setup should not be built by default as it is right now
When we build GE executable, we build the sln + native (ShellEx, AskPass; in separate solutions)
The build scripts then build Setup explicitly

The VS plugins has dependencies to GE.exe and the old VS plugin had to be included in the setup. Such dependencies must be handled explicitly in the build script if GE project is built instead of the solution.